assets/dev/js/editor/components/validator/breakpoint.js

```diff
@@ -0,0 +1,55 @@
+const NumberValidator = require( 'elementor-validator/number' );
+
+module.exports = NumberValidator.extend( {
+	breakpointKeysArray: [],
+
+	getDefaultSettings: function() {
+		return {
+			validationTerms: {
+				// Min width we allow for mobile
+				min: 320,
+				// Max width we allow in general
+				max: 5120,
+			},
+		};
+	},
+
+	validationMethod: function( newValue ) {
+		const validationTerms = this.getSettings( 'validationTerms' ),
+			errors = NumberValidator.prototype.validationMethod.call( this, newValue );
+
+		if ( _.isFinite( newValue ) ) {
+			if ( ! this.validateMinMaxForBreakpoint( newValue, validationTerms ) ) {
+				errors.push( 'Value is not between the breakpoints above or under the edited breakpoint' );
+			}
+		}
+
+		return errors;
+	},
+
+	validateMinMaxForBreakpoint: function( newValue, validationTerms ) {
+		let isValid = true;
+
+		if ( ! this.breakpointKeysArray.length ) {
+			this.breakpointKeysArray = Object.keys( elementorFrontend.config.responsive.activeBreakpoints );
+		}
+
+		const breakpointIndex = this.breakpointKeysArray.indexOf( validationTerms.breakpointName ),
```

Comment:
Use `Stylesheet.getDeviceMinBreakpoint( deviceName );` for getting the min breakpoint

Reply:
Done.